frontend/cypress/integration/common/hooks.ts

```diff
@@ -0,0 +1,9 @@
+import { Before } from '@badeball/cypress-cucumber-preprocessor';
+
+Before({tags: '@gateway-api'}, async function () {
+  cy.exec('kubectl get crd gateways.gateway.networking.k8s.io',{failOnNonZeroExit: false}).then((result) => {
+    if (result.code != 0){
+      this.skip();
```

Comment:
+1, instead of silently skipping we instead want to install the required CRDs.  If we ever do want to skip tests then it should not be done silently, we'd need to write something out to make sure it is known that tests are being skipped.